Source/common/SNTConfigurator.m

```diff
@@ -79,6 +79,7 @@ @implementation SNTConfigurator
 static NSString *const kEnableSystemExtension = @"EnableSystemExtension";
 
 static NSString *const kEnableForkAndExitLogging = @"EnableForkAndExitLogging";
+static NSString *const kSkipOtherEndpointSecurityClients = @"SkipOtherEndpointSecurityClients";
```

Comment:
_Skip_ sounds like we might be preventing them from doing processing. Maybe _ignore_?

Reply:
Good idea, done